bin/tld.js

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var tldjs = require('..');
```

Comment:
Does it import `index.js` by default?

Reply:
Yes it does import the `main` declared file of the parent path, and fallbacks on `index.js` if nothing is specified.